cmd/ci-operator-prowgen/main.go

```diff
@@ -199,26 +201,17 @@ func pruneStaleJobs(jobDir, subDir string) error {
 
 	wg := sync.WaitGroup{}
 
-	if err := jc.OperateOnJobConfigSubdir(jobDir, subDir, func(jobConfig *prowconfig.JobConfig, info *jc.Info) error {
+	for kl, vl := range c {
+		k, v := kl, vl
```

Comment:
nit: I think I prefer scoping by passing values into the go func, it's clearer IMO to this type of code

Reply:
I later discovered `k, v := k, v` also works, but this is superseded in a later commit anyway.